packages/react-components/src/components/MessageThread.tsx

```diff
@@ -712,7 +720,11 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
   useEffect(() => {
     (async () => {
       if (onLoadPreviousChatMessages) {
-        while (chatScrollDivRef.current.scrollTop <= 200 && !isAllChatMessagesLoadedRef.current) {
+        while (
+          chatScrollDivRef.current &&
+          chatScrollDivRef.current.scrollTop <= 200 &&
```

Comment:
I remember
(undefined <= 200) === false

So maybe this could be:
chatScrollDivRef.current?.scrollTop <= 200

Reply:
I thought that also but tsc complains:
![image](https://user-images.githubusercontent.com/2684369/123496905-5e36b900-d5df-11eb-89ea-cef76fca6bc5.png)